server/app/schema.py

```diff
@@ -428,9 +466,9 @@ def mutate(self, info, **kwargs):
             equipped_item.custom_set.owner_id
             and equipped_item.custom_set.owner_id != current_user.get_id()
         ):
-            raise GraphQLError("You don't have permission to edit that set.")
+            raise GraphQLError(_("You don't have permission to edit that set."))
         if stat != Stat.AP and stat != Stat.MP and stat != Stat.RANGE:
-            raise GraphQLError("Invalid stat to set exo")
+            raise GraphQLError(_("Invalid stat to set exo."))
```

Comment:
does this ever come into play? i was under the impression your choices would be locked to several checkboxes.